docs/pki.md

```diff
@@ -0,0 +1,83 @@
+# PKI for SecureDrop: problem statement
+
+## PKI: the status quo
+
+The readme sketches the following PKI scheme for use with the SecureDrop Protocol:
+
+| The party... | ...has the secret key... | ...with the public key... | ...signed by...    | ...so that...                                       |
+| ------------ | ------------------------ | ------------------------- | ------------------ | --------------------------------------------------- |
+| FPF          | _FPF<sub>SK</sub>_       | _FPF<sub>PK<sub>_         | —                  |
+| Newsroom     | _NR<sub>SK</sub>_        | _NR<sub>PK</sub>_         | _FPF<sub>SK</sub>_ | FPF can attest to an instance's legitimacy.         |
+| Journalist   | _J<sub>SK</sub>_         | _J<sub>PK</sub>_          | _NR<sub>SK</sub>_  | a source can verify a journalist's enrollment.      |
+| Journalist   | _JC<sub>SK</sub>_        | _JC<sub>PK</sub>_         | _J<sub>SK</sub>_   | the server can verify a journalist's fetching key.  |
+| Journalist   | _JE<sub>SK</sub>_        | _JE<sub>PK</sub>_         | _J<sub>SK</sub>_   | a source can verify a journalist's per-message key. |
+
+@mmaker described the need for
+
+> A section dedicated to how keys should be handled [that] will help preventing
+> errors in deployments. For example, the FPF could implement canaries to alert
+> users of potential compromises; Newsrooms should adhere to a minimal set of
+> rules of key rotation and identity validation for journalists, and
+> accountability that they still possess the key from time to time.[^1]
+
```

Comment:
(Thank you for preparing this summary - small note for completeness that one additional suggestion we had re PKI from an earlier reviewer of our work suggested a cross-signing approach between FPF and the newsroom)

Reply:
Thanks!  Added in a52d40161a7c205355769d35c4575d4bae358ee7.